D/diffutils/build_tarballs.jl

```diff
@@ -3,24 +3,27 @@
 using BinaryBuilder, Pkg
 
 name = "diffutils"
-version = v"3.7.0"
+version = v"3.8"
 
 # Collection of sources required to complete build
 sources = [
     ArchiveSource(
-        "https://ftp.gnu.org/gnu/diffutils/diffutils-3.7.tar.xz",
-        "b3a7a6221c3dc916085f0d205abf6b8e1ba443d4dd965118da364a1dc1cb3a26",
+        "https://ftp.gnu.org/gnu/diffutils/diffutils-$(version.major).$(version.minor).tar.xz",
+        "a6bdd7d1b31266d11c4f4de6c1b748d4607ab0231af5188fc2533d0ae2438fec",
     ),
     DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/diffutils-*/
+
 if [[ "${target}" == *-mingw* ]]; then
-    atomic_patch -p1 ../patches/win_signal_handling.patch
+    atomic_patch -p1 ../patches/win_sa_restart.patch
 fi
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --target=${compiler_target} --disable-dependency-tracking
```

Comment:
Where did you even find `compiler_target`?  I doesn't exists.

Reply:
hahaha

Reply:
thanks.